packages/fuels-core/src/lib.rs

```diff
@@ -162,9 +161,106 @@ pub fn pad_string(s: &str) -> Vec<u8> {
     padded
 }
 
+pub fn unzip_param_types(param_types: &[(String, ParamType)]) -> Vec<ParamType> {
+    param_types
+        .iter()
+        .map(|(_, param_type)| param_type)
+        .cloned()
+        .collect()
+}
+
+pub trait DecodeLog {
```

Comment:
Nit: usually, trait names are adjectives rather than verbs. So `DecodeLog` would be `DecodableLog`. Should we stick to this convention?